docs/expression.md

```diff
@@ -0,0 +1,135 @@
+---
+title: "Expression system of Gravitino"
+slug: /expression
+date: 2024-02-02
+keyword: expression function field literal reference
+license: Copyright 2024 Datastrato Pvt Ltd. This software is licensed under the Apache License version 2.
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+This page introduces the expression system of Gravitino. Expressions are vital component of metadata definition, through expressions, you can define default values for columns and function arguments for [function partitioning](./table-partitioning-bucketing-sort-order-indexes.md#table-partitioning), [bucketing](./table-partitioning-bucketing-sort-order-indexes.md#table-bucketing), and sort term of [sort ordering](./table-partitioning-bucketing-sort-order-indexes.md#sort-ordering) in tables.
+Gravitino expression system divides expressions into three basic parts: field reference, literal, and function. Function expressions can contain field references, literals, and other function expressions.
+
+## Field reference
+
+Field reference is a reference to a field in a table.
+The following is an example of creating a field reference expression, demonstrating how to create a reference for the `student` field.
+
+<Tabs>
+  <TabItem value="Json" label="Json">
```

Comment:
is it `Json` or `json` for `value`? I see that below in Java it is `value="java"`.

Reply:
After local testing, both can be rendered correctly.

This writing appears in other documents as well, should I change it to keep the styling consistent?

Reply:
I'm OK if it can be rendered correctly. If you want to unify them you can do it in another PR.